pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -262,6 +264,33 @@ func (ds *DataSource) getRGTAClient(ctx context.Context, region string) (resourc
 	return NewRGTAClient(cfg), nil
 }
 
+func (ds *DataSource) isMonitoringAccount(ctx context.Context, region string) (bool, error) {
```

Comment:
I'm curious - is status as a monitoring account tied to region? I would have thought an account was either a monitoring account or not.

Reply:
yea, according to https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Unified-Cross-Account.html "With Amazon CloudWatch cross-account observability, you can monitor and troubleshoot applications that span multiple accounts **within a Region**" and also "Each account can have **one sink per Region**." so the sink being the monitoring account is per region, but you can create multiple sinks.